Sming/Arch/Esp32/build.mk

```diff
@@ -37,7 +44,8 @@ endef
 DEBUG_VARS			+= ESP32_COMPILER_PATH ESP32_ULP_PATH ESP32_OPENOCD_PATH ESP32_PYTHON_PATH
 
 ifndef ESP32_COMPILER_PATH
-ESP32_COMPILER_PATH	:= $(call FindTool,tools/$(ESP32_COMPILER_PREFIX),/$(ESP32_COMPILER_PREFIX))
+include $(IDF_PATH)/tools/toolchain_versions.mk
+ESP32_COMPILER_PATH	:= $(IDF_TOOLS_PATH)/tools/$(ESP32_COMPILER_PREFIX)/$(CURRENT_TOOLCHAIN_COMMIT_DESC)-$(CURRENT_TOOLCHAIN_GCC_VERSION)/$(ESP32_COMPILER_PREFIX)
```

Comment:
Note: Installing IDF 4.3 will break previous Sming builds. This patch would probably fix that.